webpack.config.production.js

```diff
@@ -11,19 +11,22 @@ module.exports = {
   },
   optimization: {
     minimize: true,
-    minimizer: [new TerserPlugin()],
+        minimizer: [
+      new EsbuildPlugin({
+        target: 'es2020',
+        css: true,
+      }),
+    ],
   },
   module: {
     rules: [
       ...common.module.rules,
       {
-        test: /\.tsx?/,
-        use: [
-          {
-            loader: 'ts-loader',
-            options: { transpileOnly: true },
-          },
-        ],
+        test: /\.tsx?$/,
+        loader: 'esbuild-loader',
+        options: {
+          target: 'es2020',
```

Comment:
Ooh, this is a good point - yet another reason we might want extensive testing for this one. According to this comment, ES2020 (aka ES11) is more than fine, but this might of course break older stuff.